frappe/utils/__init__.py

```diff
@@ -43,6 +45,22 @@
 )
 
 
+def semver2pypi(version: str) -> Version:
+	"""Converts a semver version into a version from PyPI
+
+	A semver prerelease will be converted into a
+	prerelease of PyPI.
+	A semver build will be converted into a development
+	part of PyPI
+	:param semver.Version ver: the semver version
+	:return: a PyPI version
+	"""
+	ver = semantic_version.Version(version)
+	prerelease = ver.prerelease if ver.prerelease else ""
+	build = ver.build if ver.build else ""
+	return Version(f"{v}{prerelease}{build}")
```

Comment:
According to https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers

The implementation of `packaging` already normalizes gracefully, so we might as well drop this (and `semantic_version` as dependency, yay)